docs/conf.py

```diff
@@ -55,7 +54,7 @@
 
 # General information about the project.
 project = u'pysatModels'
-copyright = u'2020, Angeline G Burrell, Russell Stoneback, Jeffery Klenzing'
+cr_line = u'2020, Angeline G Burrell, Russell Stoneback, Jeffery Klenzing'
 author = u'Angeline G Burrell, Russell Stoneback, Jeffery Klenzing'
```

Comment:
```suggestion
cr_line = u'2020, Angeline G Burrell, Russell Stoneback, Jeff Klenzing'
author = u'Angeline G Burrell, Russell Stoneback, Jeff Klenzing'
```
Short form is preferred, or correct spelling

Reply:
Could also append the author string to cr_line to aid in maintenance

Reply:
Could you change to long correct spelling?  I don't trust my fingers.  Also like the idea of using authors in the cr_line.

Reply:
updated